lib/union_of.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+require 'active_record_union'
+
+module UnionOf
```

Comment:
I want to hire somebody to review and audit this gem before we merge.

Reply:
Shortlist:

- https://andyatkinson.com/consulting
- https://spektrhq.com/consultancy